src/nvim/path.c

```diff
@@ -2365,6 +2365,17 @@ int append_path(char *path, const char *to_append, size_t max_len)
 ///
 /// @return FAIL for failure, OK for success.
 static int path_to_absolute(const char *fname, char *buf, size_t len, int force)
+#ifdef MSWIN
+{
+  if (_fullpath(buf, fname, len) == NULL) {
+    return FAIL;
+  }
+# if defined(BACKSLASH_IN_FILENAME)
```

Comment:
I don't understand this. Isn't `BACKSLASH_IN_FILENAME` always defined if `MSWIN` is defined?

Reply:
Yes, or at least I think so.  But you need to wrap it in an ifdef because the function I am calling, `forward_slash()`, is wrapped in an ifdef.  If it's redundant, then that should be dealt with (ifdef removed from the function definition and all the calls to this function) in a separate commit.

Reply:
Oh I see what you mean.  There is already an `ifdef MSWIN` there.  Yes, then the `ifdef BACKSLASH_IN_FILENAME` is redundant.  But I still think a function call should correspond to its definition, in general.  Can of course change if you restate.

Reply:
Yeah, fair enough.